lib/rbac/filterer.rb

```diff
@@ -514,6 +517,10 @@ def apply_scope(klass, scope)
       end
     end
 
+    def apply_select(klass, scope, extra_cols)
+      scope.select(klass.arel_table[Arel.star]).select(extra_cols)
```

Comment:
instead of `apply_select` / `:extra_cols`, is it possible to just pass the `:select` all the way through?